packages/block-library/src/common.scss

```diff
@@ -6,6 +6,9 @@
 	@include background-colors-deprecated();
 	@include foreground-colors-deprecated();
 	@include gradient-colors-deprecated();
+	--wp--preset--font-size--normal: 16px;
```

Comment:
Shouldn't these be defined in the mixin instead?

Reply:
Had to remove the mixin https://github.com/WordPress/gutenberg/pull/37381#discussion_r768890350

Reply:
Got it. Let's add a comment that these are for compatibility as well.